www/content/guides/links.mdx

```diff
@@ -22,8 +22,7 @@ yarn add html-react-parser
 Update the component from [images](/docs/guides/inline-images) to handle inline links.
 
 ```tsx title=components/body.tsx
-import { HTMLReactParserOptions, domToReact } from "html-react-parser" // highlight-line
-import { Element } from "domhandler/lib/node"
+import { HTMLReactParserOptions, domToReact, Element } from "html-react-parser" // highlight-line
```

Comment:
This change doesn't look right.

I took a look at the domhandler NPM package and it now exports Element from its main entry point. See [Line 14 of src/index.ts](https://github.com/fb55/domhandler/blob/cecfec404dc45586853b8c4801dc11d754ff5bc9/src/index.ts#L14).

So the change should be:

```
import { Element } from "domhandler"
```